### PR TITLE
Editdiver

### PR DESCRIPTION
**Pull request by FlyingMoe91**

### CODE REVIEW

File: src/components/FormCreateDiver/CreateDiver.js [hunk 1]
```diff
@@ -35,8 +34,10 @@ export default function AddDive({ onClickBack, onCreate }) {
             id="certification"
             name="certification"
             type="text"
-            onChange={handleChange}
             maxLength="10"
```
Comment (Sven-Lyco) [inline src/components/FormCreateDiver/CreateDiver.js:37]: is the length of the certificate number standardized?

File: src/components/AddDiveForm/AddDive.js [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
-
   return (
-    <>
+    <Wrapper>
       <ButtonBack name="back" onClick={onClickBack}>
         <ArrowBack />
+        <ScreenReaderOnly>back to divelogs</ScreenReaderOnly>
       </ButtonBack>
       <Headline>log new dive</Headline>
       <FormStyled
```
Comment (Ju-Schn) [inline src/components/AddDiveForm/AddDive.js:15]: I suggest aria-labelledby="form-title" and id="form-title" at the Headline component. If you do this screenreader is able to translate it if website is translated 🙂 

Aria-label is not translateable (is this an english word?) 🤔  😄

File: src/pages/home.js [hunk 3]
```diff
@@ -7,20 +7,24 @@ import DiverInfo from '../components/DiverInfomation/DiverInfo';
 
 export default function Home({ diveData }) {
   const [createDiverPageActive, setCreateDiverPageActive] = useState(false);
-  const [diverInfo, setDiverInfo] = useLocalStorage('diverInfo', []);
+  const [diverInfo, setDiverInfo] = useLocalStorage('diverInfo', ['']);
```
Comment (Sven-Lyco) [inline src/pages/home.js:10]: I'm asking me if the initial state is correct 🤔

**Review by Sven-Lyco — COMMENTED** [review]
Looks good to me, just a few comments :) Great Job 🚀

File: src/components/DiverInfomation/DiverInfo.spec.js [hunk 4]
```diff
@@ -1,18 +1,17 @@
+import { userEvent } from '@storybook/testing-library';
```
Comment (Ju-Schn) [inline src/components/DiverInfomation/DiverInfo.spec.js:1]: Do you want to use the storybook testing library or was it the autocompletion? 😉

File: src/pages/Logbook.js [hunk 5]
```diff
@@ -39,8 +41,8 @@ export default function Logbook({ diveData, onDelete, onCreateDive }) {
             ) => (
               <li key={nanoid()}>
```
Comment (Ju-Schn) [inline src/pages/Logbook.js:42]: Couldn't you use the _id here?

**Review by Ju-Schn — APPROVED** [review]
Looks good to me! Just some suggestions ☺️

**Pull request merged**